docs/UserGuide.md

```diff
@@ -244,21 +244,21 @@ Format: `group [id/NUSID] [g/GROUP] [t/TAG]`
 
 Schedule a meeting with a person in the address book.
 
-Format: `schedule id/NUSID [s/SCHEDULE] [r/REMARK]`
+Format: `schedule id/NUSID [s/SCHEDULE r/REMARK]`
 
 > Note: 
 > * Schedule a meeting with a person of the specified `nusId` on the specified `schedule` with a `remark`.
 > * The `NUSID` refers to the NUSID shown in the displayed person list.
-> * The `NUSID` **must be a 7-digit number following an 'E'**
+> * The `NUSID` **must be a 7-digit number following an 'E'**.
 > * Both `schedule` and `remark` must be either provided or not provided.
 > * If `schedule` and `remark` are not provided, the schedule will be removed.
 
-**Tip:** The `schedule` must be in one of the formats: `DD/MM/YYYY`, `DD-MM-YYYY`, `DD.MM.YYYY`, `MMM DD, YYYY`, `DD MMM YYYY`.
+**Tip:** The `schedule` must be in one of the formats: `DD/MM/YYYY`, `DD-MM-YYYY`, `DD.MM.YYYY`, `MMM DD, YYYY`, `DD MMM YYYY`. Example of date formats include: `12-12-2024`, `12/12/2024`, `12.12.2024`, `Dec 12, 2024`, `12 Dec 2024`.
```

Comment:
Giving examples to MMM to address issue #175 when the user does not know the that MMM corresponds to the 3-letter abbreviation of the month .